packages/testlab/src/client.ts

```diff
@@ -33,13 +33,13 @@ export function createClientForHandler(
  * @param app A running (listening) instance of a RestApplication.
  */
 export function createRestAppClient(app: RestApplicationLike) {
-  const url = app.restServer.url;
-  if (!url) {
+  const rootUrl = app.restServer.rootUrl;
```

Comment:
Let's keep the size of this patch smaller. Also to support older version that don't expose `rootUrl` property yet, we should fall back to `url` when `rootUrl` is not set.

```suggestion
  const url = app.restServer.rootUrl || app.restServer.url;
```